go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/kuadrant/limitador-operator
 
-go 1.19
+go 1.20
 
 require (
 	github.com/go-logr/logr v1.2.4
 	github.com/google/go-cmp v0.5.9
 	github.com/onsi/ginkgo/v2 v2.11.0
 	github.com/onsi/gomega v1.27.8
-	go.uber.org/zap v1.19.1
+	go.uber.org/zap v1.24.0
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
-	k8s.io/klog/v2 v2.60.1
-	sigs.k8s.io/controller-runtime v0.12.2
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
+	k8s.io/client-go v0.26.1
+	k8s.io/klog/v2 v2.80.1
+	sigs.k8s.io/controller-runtime v0.14.6
```

Comment:
Running `go mod tidy` suggests the `go.mod` is still out of sync with the new changes.

For example, `k8s.io/utils v0.0.0-20221128185143-99ec85e7a448` is now a required dependency 🤔

Reply:
Thanks! 

I added `go mod tidy` verification test